libshaderc/include/shaderc.h

```diff
@@ -128,6 +128,10 @@ void shaderc_compile_options_set_disassembly_mode(
 void shaderc_compile_options_set_preprocessing_only_mode(
     shaderc_compile_options_t options);
 
+// Sets the compiler to suppress all warning messages before they are output.
```

Comment:
Please change to "Sets the compiler mode to suppress warnings."

There are two fixes here:
1. add "mode" to make the grammar work.
2. recognize that there are two conceptual phases: generating a warning, and outputting it.  There's a distinction in the sense that someone could have "-Werror" which changes a *generated* warning into an error, but still not print out the warning as a warning.